packages/lodestar-cli/src/lodecli/cmds/beacon/cmds/run/run.ts

```diff
@@ -0,0 +1,43 @@
+import process from "process";
+import {Arguments} from "yargs";
+import deepmerge from "deepmerge";
+import {initBLS} from "@chainsafe/bls";
+import {BeaconNode} from "@chainsafe/lodestar/lib/node";
+import {createNodeJsLibp2p} from "@chainsafe/lodestar/lib/network/nodejs";
+import {config as mainnetConfig} from "@chainsafe/lodestar-config/lib/presets/mainnet";
+import {WinstonLogger} from "@chainsafe/lodestar-utils/lib/logger";
+import defaultOptions, {IBeaconNodeOptions} from "@chainsafe/lodestar/lib/node/options";
+
+import {readPeerId, readEnr, writeEnr} from "../../../../network";
+import {IBeaconArgs} from "../../options";
+
+/**
+ * Run a beacon node
+ */
+export async function run(options: Arguments<IBeaconArgs & Partial<IBeaconNodeOptions>>): Promise<void> {
```

Comment:
Maybe we could have something similar to lighthouse:
```terminal
lodecli run bn //to run beacon node
lodecli run vn //to run validator
etc
```

Reply:
I believe we can add command aliases, will play with some of that